src/list/list.test.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import {render, cleanup, screen} from "@testing-library/react";
+import "@testing-library/jest-dom";
+import {create} from "react-test-renderer";
+
+import {testA11y} from "../core/utils/test/testUtils";
+import List, {ListProps} from "./List";
+import ListItem from "./item/ListItem";
+
+describe("<List />", () => {
+  afterEach(cleanup);

```

Comment:
Do we need to use `cleanup` (or, why do we need it)? In this [blog post](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library) it says :
> For a long time now cleanup happens automatically (supported for most major testing frameworks) and you no longer need to worry about it.

Reply:
Thank you. It seems we don't need to use cleanup. So i will remove it from all tests